### ISSUE TITLE
`git_utils.remove_file` error handling

### ISSUE THREAD

**Issue by blester125** [r-three/git-theta#104]
We should have a graceful error, like a no-op, when we try to remove a file from git that is not currently tracked. It can be hard to check if a file is tracked (existence on disk is not enough) so we should bundle this into the remove file function itself.

**Comment by nkandpa2** [comment]
No longer needed after #114
